src/services/server-translations/dataIntegrityChecks.js

```diff
@@ -0,0 +1,123 @@
+    program_rule_actions_without_stage: i18n.t(
+        'Program rule actions without stage'
+    ),
+    categories_one_default_category: i18n.t(
+        'Only one "default" category should exist'
+    ),
+    categories_unique_category_combo: i18n.t(
+        'Lists category combos that share a combination of categories with at least one other category combo'
+    ),
+    org_units_with_cyclic_references: i18n.t(
+        'Org units with cyclic references'
+    ),
+    program_rule_variables_without_data_element: i18n.t(
+        'Program rule variables without data element'
+    ),
+    validation_rules_with_invalid_right_side_expression: i18n.t(
+        'Validation rules with invalid right side expression'
+    ),
+    data_sets_not_assigned_to_org_units: i18n.t(
+        'Data sets not assigned to org units'
+    ),
+    data_elements_violating_exclusive_group_sets: i18n.t(
+        'Data elements violating exclusive group sets'
+    ),
+    org_unit_groups_without_group_sets: i18n.t(
+        'Org unit groups without group sets'
+    ),
+    program_rule_variables_without_attribute: i18n.t(
+        'Program rule variables without attribute'
+    ),
+    org_units_violating_exclusive_group_sets: i18n.t(
+        'Org units violating exclusive group sets'
+    ),
+    program_rule_actions_without_data_object: i18n.t(
+        'Program rule actions without data object'
+    ),
+    categories_one_default_category_option: i18n.t(
+        'Only one "default" category option should exist'
+    ),
+    categories_one_default_category_option_combo: i18n.t(
+        'Only one "default" category option combo should exist'
+    ),
+    org_units_without_groups: i18n.t('Org units without groups'),
+    program_indicators_without_expression: i18n.t(
+        'Program indicators without expression'
+    ),
+    indicators_violating_exclusive_group_sets: i18n.t(
+        'Indicators violating exclusive group sets'
+    ),
+    periods_duplicates: i18n.t('Periods duplicates'),
+    data_elements_assigned_to_data_sets_with_different_period_types: i18n.t(
+        'Data elements assigned to data sets with different period types'
+    ),
+    program_rules_without_action: i18n.t('Program rules without action'),
+    org_units_being_orphaned: i18n.t('Org units being orphaned'),
+    validation_rules_with_invalid_left_side_expression: i18n.t(
+        'Validation rules with invalid left side expression'
+    ),
+    indicators_with_invalid_denominator: i18n.t(
+        'Indicators with invalid denominator'
+    ),
+    program_rule_actions_without_section: i18n.t(
+        'Program rule actions without section'
+    ),
+}
+
+export const severityMap = {
+    WARNING: i18n.t('Warning'),
+    SEVERE: i18n.t('Severe'),
+}
+
+const reportTypeMap = {
+    REPORT: i18n.t('Report'),
+    SUMMARY: i18n.t('Summary'),
+    DETAILS: i18n.t('Details'),
+}
+
+const snakeCaseToHumanReadable = string => {
```

Comment:
Is this so that untranslated string will look better in english? Because I can imagine that this would also make it harder to catch untranslated strings, as we usually look at the UI in english (or I do at least).

Reply:
>This endpoint is "semi-dynamic", in the sense that it's described by a yaml-file, but still compiled at build-time, it also does not have support for translations. I've included translations for all the current jobs, but also have a simple fallback for any other jobs with a simple "snake"-case converter. This should all ideally be done on the backend, but we do what we have to work with what've got.

I think this explains it. I'll close this as it seems like we should discuss this with backend.